frontend/benefit/handler/src/components/applicationList/useApplicationList.ts

```diff
@@ -43,7 +44,10 @@ const useApplicationList = (
         companyId: company ? company.business_id : '-',
         employeeName:
           getFullName(employee?.first_name, employee?.last_name) || '-',
-        submittedAt: submitted_at ? formatDate(new Date(submitted_at)) : '-',
+        submittedAt: submitted_at ? convertToUIDateFormat(submitted_at) : '-',
+        additionalInformationNeededBy: additional_information_needed_by
```

Comment:
`convertToUIDateFormat(additional_information_needed_by) || '-'` is a bit shorter